### DISCUSSION TITLE
multilingual supported in clip4clip?

### DISCUSSION THREAD

**Discussion by yfsu73** [towhee-io/towhee#2698] [Q&A]
I just wonder if clip4clip supports multilingual? does it support by default? or I shall do any configuration? 

thanks in advance,
Steven

**Comment by junjiejiangjjj** [comment]
@zc277584121

**Comment by zc277584121** [comment]
@yfsu73  https://arxiv.org/pdf/2104.08860

**Comment by zc277584121** [comment]
here is the paper for clip4clip, it dosen't support multilingual .
